<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFaultDialog</class>
 <widget class="QDialog" name="AddFaultDialog">
  <property name="windowTitle">
   <string>Add Fault Feature</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
        <widget class="QLabel" name="labelStrike">
         <property name="text">
          <string>Strike (°)</string>
         </property>
        </widget>
      </item>
      <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="strike_input"/>
      </item>
      <item row="1" column="0">
        <widget class="QLabel" name="labelDip">
         <property name="text">
          <string>Dip (°)</string>
         </property>
        </widget>
      </item>
      <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="dip_input"/>
      </item>
      <item row="2" column="0">
        <widget class="QLabel" name="labelCentre">
         <property name="text">
          <string>Centre (X, Y, Z)</string>
         </property>
        </widget>
      </item>
      <item row="2" column="1">
        <layout class="QHBoxLayout" name="centreLayout">
         <item>
          <widget class="QDoubleSpinBox" name="centre_x_input"/>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="centre_y_input"/>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="centre_z_input"/>
         </item>
        </layout>
      </item>
      <item row="3" column="0">
        <widget class="QLabel" name="labelExtents">
         <property name="text">
          <string>Ellipsoid Extents (X, Y, Z)</string>
         </property>
        </widget>
      </item>
      <item row="3" column="1">
        <layout class="QHBoxLayout" name="extentsLayout">
         <item>
          <widget class="QDoubleSpinBox" name="extent_x_input"/>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="extent_y_input"/>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="extent_z_input"/>
         </item>
        </layout>
      </item>
      <item row="4" column="0">
        <widget class="QLabel" name="labelDisplacement">
         <property name="text">
          <string>Displacement</string>
         </property>
        </widget>
      </item>
      <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="displacement_input"/>
      </item>
      <item row="5" column="0">
        <widget class="QLabel" name="labelPitch">
         <property name="text">
          <string>Pitch (°)</string>
         </property>
        </widget>
      </item>
      <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="pitch_input"/>
      </item>
      <item row="6" column="0">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
      </item>
      <item row="6" column="1">
        <widget class="QLineEdit" name="name_input"/>
      </item>
     </layout>
    </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
